### PR TITLE
EDGE-325 virtual-camera shows strange errors

### PR DESCRIPTION
**Pull request by ionyshch-prismsl**
Made Client::Impl::uploadObjectStream() consider both 200 and 201 as success.

### CODE REVIEW

File: src/client.cpp [hunk 1]
```diff
@@ -736,7 +736,7 @@ Status Client::Impl::uploadObjectStream(id_t accountId, id_t instrumentId,
         return makeNetworkError();
     }
 
-    if (cs->getResponseCode() != 201)
+    if (cs->getResponseCode() != 201 && cs->getResponseCode() != 200)
```
Comment (vsharapov-prism) [inline src/client.cpp:739]: According to rest API all 2xx codes are success. May be better allow all range?

Comment (ionyshch-prismsl) [reply src/client.cpp:739]: But 2xx code may not correspond to operation. 200 isn't appropriate for resource creation (post), but 201 is.

Comment (vsharapov-prism) [reply src/client.cpp:739]: At this moment uploadObjectStream allows 200 & 201, but at other places, like uploadBackground only 201 allowed; looks like mess.
2xx code is success code,so it is wrong to say that errors happens if code equal for ex. 204. I think in this case better to write just warning: "Data processed but response is unusual".

Comment (ionyshch-prismsl) [reply src/client.cpp:739]: We can't say it as we do different operations: create and update with one HTTP command POST. It is broken from the very start. We can just make it work for now, by providing "hotfix"

**Review by vsharapov-prism — APPROVED** [review]

**Pull request merged**